src/pages/resultsView/fusion/Fusions.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ *
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import { ResultsViewPageStore } from '../ResultsViewPageStore';
+import { action, computed } from 'mobx';
+import { MSKTab, MSKTabs } from 'shared/components/MSKTabs/MSKTabs';
+import LoadingIndicator from '../../../shared/components/loadingIndicator/LoadingIndicator';
+import ResultsViewFusionMapper from './ResultsViewFusionMapper';
+import autobind from 'autobind-decorator';
+import ResultsViewURLWrapper from 'pages/resultsView/ResultsViewURLWrapper';
+
+export interface IFusionPageProps {
+    store: ResultsViewPageStore;
+    urlWrapper: ResultsViewURLWrapper;
+}
+
+@autobind
+@observer
+export default class Fusions extends React.Component<IFusionPageProps, {}> {
+    protected generateTabs(genes: string[]) {
```

Comment:
would be good if this was a computed getter.  this should be easy to do because you are passing it 
```this.props.store.hugoGeneSymbols!```
so you can just refernece this inside the method instead of passing it in